include/OpenColorIO/OpenColorIO.h

```diff
@@ -593,6 +593,20 @@ class OCIOEXPORT Config
     //    The argument is cloned.
     void setFileRules(ConstFileRulesRcPtr fileRules);
 
+    // !cpp:function:: Get the color space of the first rule that matched filePath.
+    const char * getColorSpaceFromFilepath(const char * filePath) const;
+
+    //!cpp:function:: Most applications will use the preceding method, but this method may be
+    // used for applications that want to know which was the highest priority rule to match
+    // filePath.  The getNumCustomKeys(ruleIndex) and custom keys methods may then be used
```

Comment:
Since this method was moved, we might want to clarify where `getNumCustomKeys(ruleIndex)` is defined. Maybe just change to `FileRules::getNumCustomKeys(ruleIndex)`?

Reply:
good idea